src/backend/base/langflow/api/v1/endpoints.py

```diff
@@ -592,7 +592,15 @@ async def custom_component_update(
         if hasattr(cc_instance, "set_attributes"):
             template = code_request.get_template()
             params = {
-                key: value_dict.get("value") for key, value_dict in template.items() if isinstance(value_dict, dict)
+                key: value_dict.get("value")
+                if value_dict.get("_input_type") != "IntInput"
+                else (
+                    int(value_dict.get("value"))
+                    if value_dict.get("_input_type") != "FloatInput"
+                    else float(value_dict.get("value"))
+                )
+                for key, value_dict in template.items()
+                if isinstance(value_dict, dict)
```

Comment:
Maybe something like this would be better
```suggestion
# Define a mapping of input types to their conversion functions
type_converters = {
    "IntInput": int,
    "FloatInput": float,
}

# Generate the params dictionary
params = {
    key: type_converters.get(value_dict.get("_input_type"), lambda x: x)(value_dict.get("value"))
    for key, value_dict in template.items()
    if isinstance(value_dict, dict)
}
```